README.md

```diff
@@ -8,6 +8,22 @@ How to use:
 * Start server: go run main.go server
 * Client call to check health status of server: go run main.go client.
 
+### Instruction for Windows Service Installation
+
+If you want to install harness-docker-runner as a service in windows please follow the bellow instructions.
+#### Installation
+* Download the msi (harness-docker-runner-svc.msi) from latest releases
+* Double click the msi to start the installation process
+* Accept the liecence, click next and finish the installation.
+#### Uninstallation
+* Search and open add or remove programs from search menu
+* search for harness-docker-runner-svc
+* click Uninstall and finish.
+#### Additional Instructions
+* Service will automatically started even if you re-started your VM
+* You can manually Stop/Start the service from service menu tab.
+* Please do not delete the service manually from the services it may cause issue in uninstallation.
```

Comment:
If customer does that, put steps to handle it

Reply:
Added  below